### PR TITLE
Fix Promscale installation

### PR DESCRIPTION
**Pull request by VineethReddy02**
# Description

Fix Promscale helm/k8s install flow as per discussion with @ramonguiu 

# Version

Which documentation version does this PR apply to?

- [ ] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #[insert issue link, if any]

### CODE REVIEW

File: promscale/installation/index.md [hunk 1]
```diff
@@ -14,10 +14,9 @@ telemetry data.
 If you have Prometheus or OpenTelemetry installed, you can install Promscale
 using one of these methods:
 
-*   Install Promscale from a [pre-built Docker container][promscale-install-docker]
-*   Install Promscale using [Helm charts][promscale-install-helm]
-*   Install Promscale using [a manifest][promscale-install-k8s-manifest]
-*   Install Promscale [from source][promscale-install-source]
+*   **Kubernetes**: install Promscale using our [Helm charts][promscale-install-helm] or using [a manifest][promscale-install-k8s-manifest].
+*   **Docker**: install Promscale from a [pre-built Docker container][promscale-install-docker].
+*   **VM or bare metal**: install Promscale [from source][promscale-install-source].
```
Comment (Loquacity) [inline promscale/installation/index.md:19]:
```suggestion
*   Install Promscale using our [Helm charts][promscale-install-helm] or using [a manifest][promscale-install-k8s-manifest].
*   Install Promscale from a [pre-built Docker container][promscale-install-docker].
*   Install Promscale [from source][promscale-install-source].
```

File: promscale/installation/kubernetes.md [hunk 2]
```diff
@@ -26,6 +26,8 @@ credentials instead it uses the same credentials that are generated during the
 `helm install`.
 </highlight>
 
+By default, `timescaledb-single` helm chart deploys the TimescaleDB in **high-availability** mode. This results in running 3 replicas of database instances which in return comsumes 3x of disk as each database instance mounts to it's own persistent volume claim (PVC). You can disable high-availability mode by changing the `timescaledb-single` helm chart `replicaCount` values to `1` in ` values.yaml`.
```
Comment (Loquacity) [inline promscale/installation/kubernetes.md:29]:
```suggestion
By default, the `timescaledb-single` helm chart deploys TimescaleDB in 
high availability mode. By default, this creates three database replicas, 
which consumes three times the amount of disk space. Each database 
instance mounts to its own persistent volume claim (PVC). You can 
disable high availability mode by changing the `timescaledb-single` 
helm chart `replicaCount` values to `1` in ` values.yaml`.
```

File: promscale/installation/kubernetes.md [hunk 3]
```diff
@@ -73,8 +75,18 @@ can provide the database URI, or specify connection parameters.
     ```bash
     helm repo update
     ```
-1.  Open the `values.yaml` configuration file, download the tobs
-    `values.yaml` from [here][tobs-values-yaml] and locate the `connection`
+1.  Create a database with name `tsdb` for Promscale data
```
Comment (Loquacity) [inline promscale/installation/kubernetes.md:78]:
```suggestion
1.  Create a database called `tsdb` for Promscale data:
```

File: promscale/installation/kubernetes.md [hunk 4]
```diff
@@ -73,8 +75,18 @@ can provide the database URI, or specify connection parameters.
...
+    ```bash
+    MASTERPOD="$(kubectl get pod -o name --namespace <namespace> -l release=<releaseName>,role=master)"
+    kubectl exec -i --tty --namespace default ${MASTERPOD} -- psql -U postgres
+    CREATE DATABASE tsdb WITH OWNER postgres;
+    ```
+1.  Capture the `postgres` user password using
```
Comment (Loquacity) [inline promscale/installation/kubernetes.md:84]:
```suggestion
1.  Capture the `postgres` user password:
```

File: promscale/installation/kubernetes.md [hunk 5]
```diff
@@ -87,16 +99,18 @@ can provide the database URI, or specify connection parameters.
...
+    OR
+
     <tab label="Connection parameters">
 
     ```yaml
     connection:
       user: postgres
       password: ""
-      host: db.timescale.svc.cluster.local
+      host: <releaseName>.<namespace>.svc.cluster.local
```
Comment (Loquacity) [inline promscale/installation/kubernetes.md:110]:
```suggestion
      host: <RELEASE_NAMEW.<NAMESPACE>.svc.cluster.local
```

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: promscale/installation/kubernetes.md (hunk 2 above)
Comment (ramonguiu) [reply promscale/installation/kubernetes.md:29]: @VineethReddy02 where will a user find the the values.yaml file? Is there some extra information we could provide to help them locate the file?

File: promscale/installation/kubernetes.md [hunk 6]
```diff
@@ -26,6 +26,13 @@ credentials instead it uses the same credentials that are generated during the
 `helm install`.
 </highlight>
 
+By default, the `timescaledb-single` helm chart deploys TimescaleDB in 
+high availability mode. By default, this creates three database replicas, 
+which consumes three times the amount of disk space. Each database 
+instance mounts to its own persistent volume claim (PVC). You can 
+disable high availability mode by changing the `timescaledb-single` 
+helm chart `replicaCount` values to `1` in ` values.yaml`.
```
Comment (ramonguiu) [inline promscale/installation/kubernetes.md:34]: @VineethReddy02 where will users find the values.yaml file?

**Review by Rajakavitha1 — APPROVED** [review]

**Review by ramonguiu — APPROVED** [review]

File: promscale/installation/kubernetes.md [hunk 7]
```diff
@@ -2,18 +2,19 @@
 You can install Promscale on Kubernetes using Helm or using a manifest file.
 
 ## Install Promscale with Helm
```
Comment (charislam) [inline promscale/installation/kubernetes.md:4]:
```suggestion
## Install Promscale with Helm
You can install Promscale using Helm charts.
```

File: promscale/installation/kubernetes.md [hunk 8]
```diff
@@ -2,18 +2,19 @@
...
-You can install Promscale using Helm charts. The Helm charts must be installed
-in this order:
-1.  Install the TimescaleDB Helm chart
-1.  Install the Promscale Helm chart
 
 Before you begin, you must have installed Helm. For more information, including
 packages and installation instructions, see the
 [Helm documentation][helm-install]
```
Comment (charislam) [inline promscale/installation/kubernetes.md:8]:
```suggestion
[Helm documentation][helm-install].
```

File: promscale/installation/kubernetes.md [hunk 9]
```diff
@@ -2,18 +2,19 @@
...
 
+You can install Promscale using Helm charts. The Helm charts must be installed
```
Comment (charislam) [inline promscale/installation/kubernetes.md:10]:
```suggestion
The Helm charts must be installed
```

File: promscale/installation/kubernetes.md [hunk 10]
```diff
@@ -26,6 +27,25 @@ credentials instead it uses the same credentials that are generated during the
 `helm install`.
 </highlight>
 
+By default, the `timescaledb-single` helm chart deploys TimescaleDB in 
+high availability mode. By default, this creates three database replicas,
```
Comment (charislam) [inline promscale/installation/kubernetes.md:31]:
```suggestion
high availability mode. This creates three database replicas, 
```

File: promscale/installation/kubernetes.md [hunk 11]
```diff
@@ -26,6 +27,25 @@ credentials instead it uses the same credentials that are generated during the
...
 </highlight>
 
+By default, the `timescaledb-single` helm chart deploys TimescaleDB in 
+high availability mode. By default, this creates three database replicas, 
+which consumes three times the amount of disk space. Each database 
+instance mounts to its own persistent volume claim (PVC). 
+
+You can 
+disable high availability mode by changing the `timescaledb-single` 
+helm chart `replicaCount` values to `1` in [`values.yaml`][timescaledb-single-values-yaml].
```
Comment (charislam) [inline promscale/installation/kubernetes.md:37]:
```suggestion
disable high availability mode by changing the value of `replicaCount`
to `1` in [`values.yaml`][timescaledb-single-values-yaml].
```

File: promscale/installation/kubernetes.md [hunk 12]
```diff
@@ -26,6 +27,25 @@ credentials instead it uses the same credentials that are generated during the
...
+
+<procedure>
+
+#### Steps to disbale TimescaleDB High-Availability
```
Comment (charislam) [inline promscale/installation/kubernetes.md:41]:
```suggestion
#### Disabling TimescaleDB high availability mode
```

File: promscale/installation/kubernetes.md [hunk 13]
```diff
@@ -26,6 +27,25 @@ credentials instead it uses the same credentials that are generated during the
...
+1.  Download the default `values.yaml` of `timescaledb-single` helm chart from [here][timescaledb-single-values-yaml].
```
Comment (charislam) [inline promscale/installation/kubernetes.md:42]:
```suggestion
1.  Download the default `values.yaml` for the `timescaledb-single` Helm chart. To get the
file, see the [`timescaledb-kubernetes` repository][timescaledb-single-values-yaml].
```

File: promscale/installation/kubernetes.md [hunk 14]
```diff
@@ -26,6 +27,25 @@ credentials instead it uses the same credentials that are generated during the
...
+1.  In `values.yaml` change the default `replicaCount` from `3` to `1`.
```
Comment (charislam) [inline promscale/installation/kubernetes.md:43]:
```suggestion
1.  In `values.yaml`, change the default `replicaCount` from `3` to `1`. 
```

File: promscale/installation/kubernetes.md [hunk 15]
```diff
@@ -26,6 +27,25 @@ credentials instead it uses the same credentials that are generated during the
...
+You can 
+disable high availability mode by changing the `timescaledb-single` 
+helm chart `replicaCount` values to `1` in [`values.yaml`][timescaledb-single-values-yaml].
+
+<procedure>
+
+#### Steps to disbale TimescaleDB High-Availability
+1.  Download the default `values.yaml` of `timescaledb-single` helm chart from [here][timescaledb-single-values-yaml].
+1.  In `values.yaml` change the default `replicaCount` from `3` to `1`. 
+1.  In the next step while installing the `timescaledb-single` helm chart, use this `values.yaml` file with `-f` flag.
```
Comment (charislam) [inline promscale/installation/kubernetes.md:44]:
```suggestion
1.  Use this `values.yaml` file with the `-f` flag when installing the `timescaledb-single` Helm chart.
    For installation instructions, see [the procedures for installing the Helm chart](#installing-the-timescaledb-helm-chart).
```

File: promscale/installation/kubernetes.md [hunk 16]
```diff
@@ -26,6 +27,25 @@ credentials instead it uses the same credentials that are generated during the
 `helm install`.
 </highlight>
 
+By default, the `timescaledb-single` helm chart deploys TimescaleDB in
```
Comment (charislam) [inline promscale/installation/kubernetes.md:30]:
```suggestion
By default, the `timescaledb-single` Helm chart deploys TimescaleDB in 
```

File: promscale/installation/kubernetes.md [hunk 17]
```diff
@@ -73,8 +93,18 @@ can provide the database URI, or specify connection parameters.
     ```bash
     helm repo update
     ```
-1.  Open the `values.yaml` configuration file, download the tobs
-    `values.yaml` from [here][tobs-values-yaml] and locate the `connection`
+1.  Create a database called `tsdb` for Promscale data:
+    ```bash
+    MASTERPOD="$(kubectl get pod -o name --namespace <namespace> -l release=<releaseName>,role=master)"
```
Comment (charislam) [inline promscale/installation/kubernetes.md:98]:
```suggestion
    MASTERPOD="$(kubectl get pod -o name --namespace <NAMESPACE> -l release=<RELEASE_NAME>,role=master)"
```

File: promscale/installation/kubernetes.md [hunk 18]
```diff
@@ -73,8 +93,18 @@ can provide the database URI, or specify connection parameters.
...
+    kubectl exec -i --tty --namespace default ${MASTERPOD} -- psql -U postgres
+    CREATE DATABASE tsdb WITH OWNER postgres;
```
Comment (charislam) [inline promscale/installation/kubernetes.md:100]: Should these be made into separate steps?

File: promscale/installation/kubernetes.md (hunk 13 above)
Comment (ryanbooz) [reply promscale/installation/kubernetes.md:42]: I updated this some @charislam. We aren't actually directing them to the repository so the second sentence didn't make sense and this is just direct. "download the file..."

File: promscale/installation/kubernetes.md (hunk 18 above)
Comment (ryanbooz) [reply promscale/installation/kubernetes.md:100]: I don't think so @VineethReddy02, correct? However, as shown, I believe we'll need the `\` at the end of each line.

Essentially, the "MASTERPOD" is a variable passed into the next command, and the `psql` command what we're asking the `kubectl exec` to run.

But again, if someone copied and pasted these three separate lines, it wouldn't work without the `\` on the first two lines to signal continuation.

File: promscale/installation/kubernetes.md [hunk 19]
```diff
@@ -73,8 +93,18 @@ can provide the database URI, or specify connection parameters.
...
+    ```
+1.  Capture the `postgres` user password:
+    ```bash
+    echo $(kubectl get secret --namespace default tobs-credentials -o jsonpath="{.data.PATRONI_SUPERUSER_PASSWORD}" | base64 --decode)
+    ```
+1.  Download the Promscale
+    `values.yaml` from [here][promscale-values-yaml], open the `values.yaml` configuration file, and locate the `connection`
```
Comment (ryanbooz) [inline promscale/installation/kubernetes.md:107]: @charislam @Rajakavitha1 - whatever we choose to do for linking/wording on the TimescaleDB procedure above, we need to do the same thing here. I personally don't mind what @VineethReddy02 did here (linking to the word 'here'), but it's not consistent with the changes above. Decide on one of the two.

**Review by ryanbooz — CHANGES_REQUESTED** [review]
A few final comments/requests for changes.

Once these are fixed, ping me in slack tomorrow and we can make sure this goes live.

File: promscale/installation/kubernetes.md (hunk 19 above)
Comment (Rajakavitha1) [reply promscale/installation/kubernetes.md:107]: Hi @VineethReddy02  and @ryanbooz  Here's what the style guide recommends: 
"Write unique, descriptive link text that makes sense without the surrounding text. Don't use phrases such as this document, this article, or click here."

File: promscale/installation/kubernetes.md [hunk 20]
```diff
@@ -2,18 +2,19 @@
...
+in this order:
+1.  Install the TimescaleDB Helm chart
+1.  Install the Promscale Helm chart
+
 ### Install the TimescaleDB Helm chart
 Before you install the TimescaleDB Helm chart, you need to configure these
-settings in the `values.yaml` configuration file:
+settings in the [`values.yaml`][timescaledb-single-values-yaml] configuration file:
 *   Credentials for the superuser, admin, and other users
 *   TLS Certificates
```
[the remark this answers is not in the document]
Comment (ramonguiu) [reply promscale/installation/kubernetes.md:20]: @VineethReddy02 can you take a look at this?

File: promscale/installation/kubernetes.md (hunk 18 above)
Comment (ramonguiu) [reply promscale/installation/kubernetes.md:102]: That's my understanding as well. @VineethReddy02 please confirm.

File: promscale/installation/kubernetes.md (hunk 19 above)
Comment (ramonguiu) [reply promscale/installation/kubernetes.md:109]: What about something like

1.  [Download the Promscale `values.yaml` file][promscale-values-yaml] and edit  the `connection` section in that file with your TimescaleDB connection details:

**Review by Rajakavitha1 — APPROVED** [review]

File: promscale/installation/kubernetes.md (hunk 19 above)
Comment (Rajakavitha1) [reply promscale/installation/kubernetes.md:109]: > What about something like
> 
> 1. [Download the Promscale `values.yaml` file][promscale-values-yaml] and edit  the `connection` section in that file with your TimescaleDB connection details:

Comment (Rajakavitha1) [reply promscale/installation/kubernetes.md:109]: I made a suggestion. Please feel free to make or ignore the change we can always work on editing after the content is published.

Comment (charislam) [reply promscale/installation/kubernetes.md]: Applied your suggestion @Rajakavitha1, I think it's consistent now

File: promscale/installation/kubernetes.md (hunk 18 above)
Comment (VineethReddy02) [reply promscale/installation/kubernetes.md:101]: I have slightly updated these instructions and I have tested them from my end, works fine. These instructions are taken from here in upstream [timescaledb helm docs](https://github.com/timescale/timescaledb-kubernetes/tree/master/charts/timescaledb-single#connecting-from-inside-the-cluster) and they do not need `\` explicitly at the end of each instruction.

File: promscale/installation/kubernetes.md (hunk 20 above)
[the remark this answers is not in the document]
Comment (VineethReddy02) [reply promscale/installation/kubernetes.md]: I have added a commit, that references these sections in `values.yaml`.

**Review by ryanbooz — APPROVED** [review]

**Pull request merged**